core/services/offchainreporting/oracle.go

```diff
@@ -93,7 +93,7 @@ func (d jobSpawnerDelegate) ServicesForSpec(spec job.SpecDB) (services []job.Ser
 		contractCaller,
 		d.ethClient,
 		d.logBroadcaster,
-		concreteSpec.ID,
+		jobSpec.ID,
```

Comment:
this was probably causing some subtle issues also

Reply:
How was this ever working

Reply:
Actually, this parameter does nothing at all from the looks of things. Can you remove it please?

Reply:
it is used in OCRContractConfigSubscription - specifically https://github.com/smartcontractkit/chainlink/blob/d680022a69eb93f4bb4cff867e3a587a1b68c961/core/services/offchainreporting/contract_config_subscription.go#L132

Reply:
Got it.